bazel/scripts/run_integ_tests.sh

```diff
@@ -345,8 +346,8 @@ RETRY_ATTEMPTS=2
 RERUN_PREVIOUSLY_FAILED="false"
 FAILED_LIST=()
 FAILED_LIST_FILE="/tmp/last_failed_integration_tests.txt"
-INTEGTEST_REPORT_FOLDER="/var/tmp/test_results"
-MERGED_REPORT_FOLDER="${INTEGTEST_REPORT_FOLDER}/integtest_merged_report"
+MERGED_REPORT_FOLDER="/var/tmp/test_results/"
```

Comment:
minor NIT: the last "/" is not needed (and also not in the sudo test script)

Reply:
Fixed.